### PR TITLE
chore: update dependencies in ng2 blueprint

### PR DESCRIPTION
**Pull request by beeman**
All package are updated to the latest version, except:

- protractor, as version 5 is not supported on Node 4
- typescript, as version 2.1 is not compatible with @angular/compiler-cli

### CODE REVIEW

File: packages/angular-cli/blueprints/ng2/files/package.json [hunk 1]
```diff
@@ -13,35 +13,34 @@
...
+    "rxjs": "^5.0.3",
+    "zone.js": "^0.7.6"
   },
   "devDependencies": {
-    "@angular/compiler-cli": "^2.3.1",
-    "@types/jasmine": "2.5.38",
-    "@types/node": "^6.0.42",
+    "@angular/compiler-cli": "^2.4.4",
+    "@types/jasmine": "2.5.41",
+    "@types/node": "^7.0.0",
```
Comment (filipesilva) [inline packages/angular-cli/blueprints/ng2/files/package.json]: It's better to keep this one on 6, as it's LTS but 7 isn't yet.

File: packages/angular-cli/blueprints/ng2/files/package.json [hunk 2]
```diff
@@ -13,35 +13,34 @@
...
-    "karma-jasmine": "^1.0.2",
-    "karma-remap-istanbul": "^0.2.1",
+    "karma-jasmine": "^1.1.0",
+    "karma-remap-istanbul": "^0.4.0",
     "protractor": "~4.0.13",
-    "ts-node": "1.2.1",
-    "tslint": "^4.3.0",
+    "ts-node": "2.0.0",
+    "tslint": "^4.3.1",
     "typescript": "~2.0.3"
```
Comment (filipesilva) [inline packages/angular-cli/blueprints/ng2/files/package.json]: Can you do TS ~2.0.10?

File: packages/angular-cli/blueprints/ng2/files/package.json [hunk 3]
```diff
@@ -13,35 +13,34 @@
...
+    "@angular/platform-browser-dynamic": "^2.4.4",
+    "@angular/router": "^3.4.4",
     "core-js": "^2.4.1",
-    "rxjs": "^5.0.1",
-    "ts-helpers": "^1.1.1",
-    "zone.js": "^0.7.2"
+    "rxjs": "^5.0.3",
+    "zone.js": "^0.7.6"
   },
   "devDependencies": {
```
Comment (filipesilva) [inline packages/angular-cli/blueprints/ng2/files/package.json]: Can you make all devDeps except `@angular/compiler-cli` use `~`?

**Comment by filipesilva** [comment]
@hansl are there any deps we shouldn't update on new projects aside from TS and protractor?

**Comment by beeman** [comment]
@hansl the failing build was caused by a breaking change in `jasmine-spec-reporter`: 

> jasmine-spec-reporter module exports SpecReporter and DisplayProcessor, see SpecReporter and DisplayProcessor usages.

https://github.com/bcaudan/jasmine-spec-reporter/releases/tag/v3.0.0

**Pull request closed without merging**